lib/parser.js

```diff
@@ -347,6 +347,12 @@ module.exports = function parser(identifier, string) {
         expect(Token.Type.END_PAREN)
       }
 
+      if (peek().type == Token.Type.DOT) {
+        expect(Token.Type.DOT)
+        token = expect(Token.Type.WORD)
+        name = token.content
```

Comment:
we could also make it a flat structure, so something like `options['other.field.field_name'] = 'special';`